rebasebot/bot.py

```diff
@@ -129,7 +131,7 @@ def _add_to_rebase(commit_message, source_repo, tag_policy):
         if commit_tag.isnumeric():
             return not _is_pr_merged(int(commit_tag), source_repo)
 
-        raise Exception(f"Unknown commit message tag: {commit_tag}")
+        raise builtins.Exception(f"Unknown commit message tag: {commit_tag}")
```

Comment:
I would suggest introducing a more specific exception type... Throwing `Exception` is not the best practice.

This might be done in follow up though.

Reply:
This was to make the linter happy. I didn't want to make this PR grow too much out of context, but I agree with you, let's make a follow up PR to make them more specific. If you have good ideas on what you would put in those feel free to open up a PR and I'm happy to review it :)

Reply:
+1

> If you have good ideas on what you would put in those feel free to open up a PR and I'm happy to review it

in general this is similar to making a custom type from a string in go, we just want to make a new type for our custom exception so that we know if we created the error or if some other thing did.